### ISSUE TITLE
Deprecation warnings on setCustomProps and getResources

### ISSUE THREAD

**Issue by Derpalus** [thegrizzlylabs/sardine-android#25]
Hello!

I'm getting deprecation warnings on setCustomProps and getResources in OkHttpSardine.java. Apparently, the functions they override have been deprecated in Sardine.

**Comment by guillaume-tgl** [comment]
Yes, as mentioned in the comments, it's better to use `list` instead of `getResources` and `patch` instead of `setCustomProps`.

**Comment by Derpalus** [comment]
Of course, however, I was thinking about Android Studio Lint complaining about deprecation and even failing release compilations if these warnings remain (even if not actually using the functions). A simple way of fixing them is to just mark the functions in question as deprecated in turn.

**Comment by guillaume-tgl** [comment]
Ok, I understand. This comes from the legacy Sardine library so I've never really thought about it. I'm not against removing these deprecated methods, both in the interface and in the class, or simply annotate them as deprecated. Do you want to make a PR?

**Comment by Derpalus** [comment]
Well, I'm currently just using this project in an app, I don't have it checked out or am particularly good at using GitHub, so perhaps it's easier if you do it so it gets done correctly.
